book/http.md

```diff
@@ -210,7 +210,7 @@ fact, one of the first two browsers had a [line-mode UI][line-mode].
 The server's response
 =====================
 
-The server's response starts with the line below:
+The server's response starts with the line in Figure 3.
```

Comment:
Change figure 3 to `{.center}` instead of `{.print-only}`. Same for figure 2 and figure 6.

Reply:
Figures 4 and 5 also need to be surrounded by `:::`

Reply:
I think this isn't quite the right fix for 2/3/6 (these are "annotated code" blocks, on the web we can use the HTML form instead of the image) but I fixed them to show the caption properly and only on the web, which they did do wrong.